### DISCUSSION TITLE
ファイルの時間問題

### DISCUSSION THREAD

**Discussion by teddy8997** [proshunsuke/colmsg#43] [Q&A] [answered]
すみません、一つ疑問があります。メッセージアップリのメッセージ時間とファイルの<日付>時間差は8時です。問題の原因を教えてください。

**Comment by proshunsuke** [comment, marked as answer]
colmsgではメッセージアプリから取得出来るデータを元にファイルの日付を決定しています。そのデータに登録されている日付はタイムゾーンUTC+0000で登録されています。

Android端末やiPhone端末上で表示される日付は、メッセージアプリから取得出来るデータを端末の日付に変換して表示しています。
例えば私の場合ですと、日本に住んでいるためタイムゾーンはJST、つまりUTC+0900なため9時間の時間差があります。

@teddy8997 さんの住んでいる地域のタイムゾーンはおそらくCSTでしょうか？つまりUTC+0800なため8時間の時間差があると思われます。

**Reply by teddy8997** [reply]
返事ありがとうございます。そうなんだ！分かりました。確かに台湾で住んでいる、台湾はUTC+0800です。
勉強になりました、説明が分かりやすいです。ありがとうございます。
